parsers/studies_parser.py

```diff
@@ -0,0 +1,45 @@
+from .parser import Parser
+import pandas as pd
+import json
+
+
+def xml_element_to_name(xml_element: str) -> str:
+    """Converts XML element to name.
```

Comment:
why XML? please explain

Reply:
Their HTTP replies carry an XML (HTML) element in the name of the file (carrying  an internal link to the specific study).
To make sure we only get the study name I copy just the study name.